Pages/page3.html

```diff
@@ -0,0 +1,165 @@
+              <!-- <li><a href="users/sign_in">Login</a></li> -->
+              <li><a href="page2.html">Attractions</a></li>
+              <li><a href="page3.html">About us</a></li>
+          </ul>
+        </div><!-- /.navbar-collapse -->
+      </div><!-- /.container-fluid -->
+    </nav>
+
+    <div class="jumbotron">
+      <div class="container">
+        <h1>Meet Our Travel Agents</h1>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="Thumbnail col-md-4">
+        <img src="/Users/tech-a25/desktop/eng-6-coursework/core/week-1/sparta-global-mini-website/Images/Usermale.jpg" width="250px" alt="">
+      <div class="caption">
+        <h3>Bob Smith</h3>
+        <ul>
+          <li>Manager</li>
+          <li>Employed: June 2015 - present</li>
+        </ul>
+      </div>
+      </div>
+
+      <div class="Thumbnail col-md-4">
+        <img src="/Users/tech-a25/desktop/eng-6-coursework/core/week-1/sparta-global-mini-website/Images/Usermale.jpg" width="250px" alt="">
+      <div class="caption">
+        <h3>Chris Cross</h3>
+        <ul>
+          <li>Travel Agent</li>
+          <li>Employed: March 2016 - present</li>
+        </ul>
+      </div>
+      </div>
+
+      <div class="Thumbnail col-md-4">
+        <img src="/Users/tech-a25/desktop/eng-6-coursework/core/week-1/sparta-global-mini-website/Images/Userfemale.ico" width="250px" alt="">
+      <div class="caption">
+        <h3>Elaine Scott</h3>
+        <ul>
+          <li>Travel Agent</li>
+          <li>Employed: April 2017 - present</li>
+        </ul>
+      </div>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="Thumbnail col-md-4">
+        <img src="/Users/tech-a25/desktop/eng-6-coursework/core/week-1/sparta-global-mini-website/Images/Userfemale.ico" width="250px" alt="">
+      <div class="caption">
+        <h3>Jane John</h3>
+        <ul>
+          <li>Customer Service Assistant</li>
+          <li>Employed: October 2015 - present</li>
+        </ul>
+      </div>
+      </div>
+
+      <div class="Thumbnail col-md-4">
+        <img src="/Users/tech-a25/desktop/eng-6-coursework/core/week-1/sparta-global-mini-website/Images/Usermale.jpg" width="250px" alt="">
+      <div class="caption">
+        <h3>Jason Hi</h3>
+        <ul>
+          <li>Travel Agent</li>
+          <li>Employed: October 2017 - present</li>
+        </ul>
+      </div>
+      </div>
+
+      <div class="Thumbnail col-md-4">
+        <img src="/Users/tech-a25/desktop/eng-6-coursework/core/week-1/sparta-global-mini-website/Images/Userfemale.ico" width="250px" alt="">
+      <div class="caption">
+        <h3>Jen Pen</h3>
+        <ul>
+          <li>Customer Service Assistant</li>
+          <li>Employed: January 2018 - present</li>
+        </ul>
+      </div>
+      </div>
+    </div>
+
+    <div class="pagination">
+      <a href="/Users/tech-a25/desktop/eng-6-coursework/core/week-1/sparta-global-mini-website/Pages/page2.html">&laquo;</a>
```

Comment:
Because you've used a complete file path from your local computer, these links won't actually work on my machine - change these to relative file paths.